### PR TITLE
set default intervals to 10 in config

### PR DESCRIPTION
**Pull request by bhavyalatha26**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1734)

### CODE REVIEW

**Comment by bhavyalatha26** [comment]
test!

File: src/main/resources/beam-template.conf [hunk 1]
```diff
@@ -26,7 +26,7 @@ beam.agentsim.lastIteration="int | 0"
 beam.agentsim.startTime="00:00:00"
 beam.agentsim.endTime="30:00:00"
 beam.agentsim.scheduleMonitorTask.initialDelay = 1
-beam.agentsim.scheduleMonitorTask.interval = 30
+beam.agentsim.scheduleMonitorTask.interval = 10
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: undo

File: src/main/resources/beam-template.conf [hunk 2]
```diff
@@ -231,7 +231,7 @@ beam.warmStart.path = ""
 # Debugging
 ##################################################################
 beam.debug.debugEnabled = false
-beam.debug.agentTripScoresInterval = 0
+beam.debug.agentTripScoresInterval = 10
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: undo

File: src/main/resources/beam-template.conf [hunk 3]
```diff
@@ -404,14 +404,14 @@ beam.outputs.generalizedLinkStats.startTime = 25200
...
-beam.outputs.writePlansInterval = "int | 0"
+beam.outputs.writePlansInterval = "int | 10"
 beam.outputs.writeEventsInterval = "int | 1"
 beam.outputs.writeLinkTraversalInterval = "int | 0"
-beam.physsim.writeEventsInterval = "int | 0"
-beam.physsim.writePlansInterval = "int | 0"
-beam.physsim.linkStatsWriteInterval = "int | 0"
+beam.physsim.writeEventsInterval = "int | 10"
+beam.physsim.writePlansInterval = "int | 10"
+beam.physsim.linkStatsWriteInterval = "int | 10"
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: set beam.physsim.writeMATSimNetwork = true

**Pull request closed without merging**